motorClass.py

```diff
@@ -0,0 +1,35 @@
+from componentClass import Component
+import time
```

Comment:
Here are several libraries that are not being used. I overlooked some of the previous pull requests but I would recommend to run pylint on all files for good formatting.